addons/mail/static/src/owl/components/composer/composer.js

```diff
@@ -148,6 +141,7 @@ class Composer extends Component {
      * Focusout the composer.
      */
     focusout() {
+        this._saveComposerContent();
```

Comment:
Is ```_saveComposerContent``` not explicit enough ?
About composer text input, what would be the gain to make it use the method itself instead of composer ?

Reply:
The gain from doing it in composer text input is that you don't need to pass 3 params to `storeDispatch` that each starts with `this._textInputRef.comp.`.

In fact, this looks like saving text content of of composer, so IMO makes sense for composer text input to handle this.

Reply:
Done